GliaWidgets/Component/Badge/BadgeStyle.swift

```diff
@@ -26,4 +26,29 @@ public struct BadgeStyle {
         self.fontColor = fontColor
         self.backgroundColor = backgroundColor
     }
+
+    /// Apply badge remote configuration
+    public mutating func applyBadgeConfiguration(_ badge: RemoteConfiguration.BadgeStyle?) {
+        if let font = UIFont.convertToFont(font: badge?.font) {
```

Comment:
Looks like you can use `map` function 
```suggestion
        UIFont.convertToFont(font: badge?.font).map { font = $0 }
```

Reply:
Fixed.